README.md

```diff
@@ -88,22 +97,36 @@ Please be respectful of the people who have created these resources. If you want
 * [Open ChemE](https://opencheme.github.io/) - open resources compiled at the University of British Columbia.
 * [TeachOpenCADD](https://projects.volkamerlab.org/teachopencadd/index.html) - a teaching platform for computer-aided drug design by the Volkamer Lab.
 * [Visual Encyclopedia of Chemical Engineering Equipment](https://encyclopedia.che.engin.umich.edu/) - from the University of Michigan. Articles, photographs, drawings, animations, and videos that demonstrate what equipment looks like and how it works.
+* [DCEKit](https://datachemeng.com/dcekit/) - Data Chemical Engineering toolKit
 
 ## Software and Packages
 * [ChemEng Solved](https://github.com/asanet/chemeng-solved) - Classic problems in chemical engineering solved with matlab.
 * [Chemics](https://github.com/wigging/chemics) - Python package for chemistry and chemical engineering applications.
+* [ht](https://github.com/CalebBell/ht) - Python package for heat transfer in chemical engineering design.
```

Comment:
I think ChEDL is a good addition. It looks like there's one more ChEDL package actively maintained that's missing: https://github.com/CalebBell/chemicals. Could you add this as well?

Reply:
yeah i will add it along with them